index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
```

Comment:
According to the Microverse Linters validation, the `lang` attribute is missing, please include it in the `HTML` tag.